package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.57",
+  "version": "1.1.58",
```

Comment:
ah I didn't do this properly 😞

going to put up a PR to try and fix it. sorry @roblevintennis! n00b mistake